### ISSUE TITLE
user image in profile in wrong place

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-product#1078]
it's here:
![Screenshot from 2019-10-18 13-34-49](https://user-images.githubusercontent.com/8178179/67091295-918b3880-f1ac-11e9-8a33-fc1222b6959c.png)

and used to be here:
![Screenshot from 2019-10-18 13-36-52](https://user-images.githubusercontent.com/8178179/67091308-96e88300-f1ac-11e9-8c12-bfcce674df90.png)

**Comment by fuzzylogic2000** [comment]
@phillimorland Do you have an idea?

**Comment by phillimorland** [comment]
yes! its the thing from ae that I added for the info to go into tab at small screens, I can take a look
